index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
That's quite something! Here it is reformatted:

```javascript
var at = S.at =
R.curry(R.compose(R.map(R.head),
                  R.useWith(R.uncurryN(2,
                                       R.converge(S.slice,
                                                  R.identity,
                                                  R.ifElse(R.eq(-1),
                                                           R.always(-0),
                                                           R.inc))),
                            R.identity,
                            R.identity)));
```

Sometimes a lambda is just the ticket. :)